docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -287,7 +287,7 @@ code: |
   for person in people_variables.true_values().union(built_in_people_vars_used):
     temp_people_quantity_question.append({"How many **" + person + "** can appear on this form?": "people_quantities['" + person + "']",
       "datatype": "radio",
-      "choices": [{'Exactly 1': 'one'},{'One or more':'more'}]
+      "choices": [{'Exactly 1': 'one'},{'At least one':'more'},{'Any number, including 0': 'any'}]
```

Comment:
Minor, but I think "Exactly 1",  "One or more", and "Any number, including 0" read the best